site/content/blogs/release-v120.en.md

```diff
@@ -18,7 +18,12 @@ Copilot v1.20 brings several new features and improvements:
 
 * **Environment manifests**: You can now create and update environments with a [manifest file](../docs/manifest/environment.en.md) bringing all the benefits of infrastructure as code to environments. 
    [See detailed walkthrough](#environment-manifest) for how to migrate your existing environments.
-
+* **Additional policy to build role**: You can now specify additional policy for CodeBuild Build Project Role through pipeline manifest field `additional_policy`.
+  [See detailed walkthrough](../docs/manifest/pipeline.en.md) for how to specify an additional policy document to add to the build project role. [(#3709)](https://github.com/aws/copilot-cli/pull/3709)
+* **`job run`**: Command now can be used to execute scheduled jobs.
+  [(#3692)](https://github.com/aws/copilot-cli/pull/3692)
+* **Deny default sec group**: This change adds option to deny EnvironmentSecurityGroup which is applied by default on copilot service
```

Comment:
```suggestion
* **Deny default security group**: This change adds the option to deny other services in an environment access to your service.
```
maybe this sounds better? (tbh i don't know the feature super well so if i'm wrong pls ignore me)

Reply:
Perhaps the following?
```suggestion
* **Deny default sec group**: Add an option `deny_default` to `security_groups` in service manifests to remove the EnvironmentSecurityGroup ingress that is applied by default.
```
It'd be awesome if we can link to the manifest fields here

Reply:
I don't see any changes in the docs/manifest to refer these flags to the manifest fields on our docs.